src/applications/static-pages/homepage-hero-randomizer/sass/homepage-hero.scss

```diff
@@ -0,0 +1,48 @@
+/*****
+
+  New homepage hero banner with a CTA to Sign In. This component displays 3 out of 8 Veteran portraits, randomly chosen on page load.
+
+  TODO:
+  - refactor vets-website/src/platform/site-wide/sass/modules/_m-homepage.scss
+
+  Date: April 2022
+  Author: allison@cityfriends.tech
+
+******/
+
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.homepage-hero {
+  // gradient background from left to right
+  &__wrapper {
+    background: linear-gradient(to right, $color-old-browser-background-start 37%, $color-old-browser-background-end);
```

Comment:
Mostly guesstimating since the inspector on Sketch isn't precise. This screenshot was taken from [RT's Sketch file](https://www.sketch.com/s/3aa40506-4be2-46cc-876b-93f1a9f3a857/a/v8Ag388#Inspector). Notice how there is a slight linear gradient from left to right across the whole background color of the hero (you can also see it by inspecting it). 

When setting leaving it as default (0% to 100%), the change in gradient color occurs at 50%, but it seems like the color change is happening more to the left.

<img width="1292" alt="Screen Shot 2022-04-21 at 2 20 54 PM" src="https://user-images.githubusercontent.com/8542413/164529865-9e7aaf26-10f7-4ae6-80d8-868514ff6b34.png">